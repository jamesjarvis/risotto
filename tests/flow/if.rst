if true {
    println(1)
}

if false {
    println(2)
}


if true {
    println(3)
} else {
    println(4)
}

if false {
    println(5)
} else {
    println(6)
}

if true {
    println(7)
    if false {
        println(8)
    } else {
        println(9)
        if true {
            println(10)
        }
    }
    println(11)
}

if false {
    println(12)
} else if false {
    println(14)
} else if false {
    println(15)
} else {
    println(16)
}

if true {
    println(17)
} else if false {
    println(18)
} else if false {
    println(19)
} else {
    println(20)
}

if false {
    println(21)
} else if false {
    println(22)
} else if true {
    println(23)
} else if false {
    println(24)
} else {
    println(25)
}

if false {
    println(26)
} else if false {
    println(27)
}

if false {
    println(28)
} else if true {
    println(29)
}

println("done")
