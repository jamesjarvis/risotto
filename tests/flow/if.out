1
3
6
7
9
10
11
16
17
23
29
done
